language/stdlib/modules/LBR.move

```diff
@@ -150,6 +150,7 @@ module LBR {
     public fun calculate_component_amounts_for_lbr(amount_lbr: u64): (u64, u64)
     acquires Reserve {
         LibraTimestamp::assert_operating();
+        assert(exists<Reserve>(CoreAddresses::LIBRA_ROOT_ADDRESS()), Errors::not_published(ERESERVE));
```

Comment:
Is there perhaps a global invariant missing that Reserve exists after genesis? Because you already assert_operating, this may become obsolete. General picture is that if EXECUTION_FAILURE can provenly not happen, we also do not need to catch it. As long as we do not use partial aborts spec, we should always catch if this does not happen.

Reply:
You are right. There is already a global invariant there and we don't need this assertion anymore.